### PR TITLE
Changed the pytest and reportportal version to 6.2.2 and 5.0.8

### PR DESCRIPTION
**Pull request by Annapooraniqxf2**
Created this pull request to fix the issue while running the test and log the result in reportportal.
The error I got while running this command is pytest tests/test_example_form --reportportal
** "INTERNALERROR> AttributeError: module '_pytest.logging' has no attribute 'get_actual_log_level'"**
To fix this issue I have upgraded the reportportal and pytest to recent version
Pytest-6.2.2
Reportportal-5.0.8

Run pip install -r requirements.txt

To fix the issue:
1. Created the pytest fixture for report portal in conftest.py

@pytest.fixture
def reportportal_service(request):
"pytest service fixture for reportportal"
reportportal_pytest_service = None
try:
if request.config.getoption("--reportportal"):
reportportal_pytest_service = request.node.config.py_test_service
except Exception as e:
print("Exception when trying to run test: %s"%file)
print("Python says:%s"%str(e))
return reportportal_pytest_service

2. Checing whether we pass the reportportal or not , if yes calling the set_rp_logger

if reportportal_service:
test_obj.set_rp_logger(reportportal_service)

We set the rp_logger if we pass reportportal in test.

3. If rp_logger is not none then in Base_page then we call the save_screenshot_reportportal and images are getting attached in reportportal.

4. Files changed are:
1. Conftest.py
2. Base_Page.py
3. Base_logging.py

5. Add a new file reportportol_conf.py and add these lines. File to be added in the git ignore also.
report_portal_uuid = "reportportal uuid"
report_portal_endpoint = "reportportal endpoint"
Please ask me these details before you do the code review.

6. To run the test
pytest test_example_form.py --reportportal

7. How did I test it
I ran the test with and without reportportal and I can see the attachment and info in reportportal.

### CODE REVIEW

**Review by shivahari — APPROVED** [review]

File: conftest.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from page_objects.PageFactory import PageFactory
 from conf import browser_os_name_conf
 from conf import base_url_conf
+from conf import report_portol_conf
```
Comment (qxf2) [inline conftest.py]: @Annapooraniqxf2  - you have a typo. Its `portal` not `portol`. Can you please fix? You can use `git mv` to rename the file.

File: conftest.py [hunk 2]
```diff
@@ -401,10 +402,10 @@ def pytest_configure(config):
     if_reportportal =config.getoption('--reportportal')
 
     try:
-        config._inicache["rp_uuid"]="c896cbb7-d8f0-4cd0-a0f7-18366057af21"
-        config._inicache["rp_endpoint"]="http://18.221.13.146:8080"
+        config._inicache["rp_uuid"] = report_portol_conf.report_portal_uuid
+        config._inicache["rp_endpoint"]= report_portol_conf.report_portal_endpoint
         config._inicache["rp_project"]="default_personal"
-        config._inicache["rp_launch"]="default_TEST_EXAMPLE"
+        config._inicache["rp_launch"]="Qxf2 Page object model launch"
```
Comment (qxf2) [inline conftest.py:408]: This and the rp_project values should come from the conf.

File: .gitignore [hunk 3]
```diff
@@ -12,3 +12,4 @@ geckodriver.log
 debug.log
 __pycache__
 venv*
+
```
Comment (qxf2) [inline .gitignore:15]: Any reason you have introduced an extra line here?

File: conftest.py (hunk 1 above)
Comment (Annapooraniqxf2) [reply conftest.py]: Fixed it

File: conftest.py (hunk 2 above)
Comment (Annapooraniqxf2) [reply conftest.py:408]: Fixed it

File: .gitignore (hunk 3 above)
Comment (Annapooraniqxf2) [reply .gitignore:15]: Fixed it

**Pull request merged**